packages/layer-selector/LayerSelectorWithMap.stories.js

```diff
@@ -0,0 +1,49 @@
+import React, { useRef, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { LayerSelector, LayerSelectorContainer } from './LayerSelector';
+import { ModulesHelper } from '../../test-helpers/storyHelpers';
+
+
+export default {
+  title: 'LayerSelector/WithMap',
+  decorators: [ModulesHelper]
+};
+
+const WithMap = ({ modules, scale, zoom, center=[-112, 40] }) => {
+  const mapDiv = useRef();
+  useEffect(() => {
+    const map = new modules.Map();
+    const view = new modules.MapView({
+      container: mapDiv.current,
+      map,
+      center,
+      zoom,
+      scale
+    });
+    const selectorNode = document.createElement('div');
+    view.ui.add(selectorNode, 'top-right');
+
+    const layerSelectorOptions = {
+      view: view,
+      quadWord: process.env.QUAD_WORD,
+      baseLayers: ['Hybrid', 'Lite', 'Terrain', 'Topo', 'Color IR'],
+      overlays: ['Address Points'],
+      modules
+    };
+
+    ReactDOM.render(
+      <LayerSelectorContainer>
+        <LayerSelector {...layerSelectorOptions}></LayerSelector>
+      </LayerSelectorContainer>,
+      selectorNode);
+  }, [modules, mapDiv, center, zoom, scale]);
+
+  return (
+    <div ref={mapDiv} style={{ position: "absolute", top: 0, left: 0, bottom: 0, right: 0 }}></div>
+  );
+};
+
+export const zoom = (modules) => <WithMap modules={modules} zoom={6} />;
```

Comment:
I think these would add more value if they were unit tests instead of stories. What are your thoughts on that?

Reply:
I think that it's nice to see how the map behaves in each of these situations. But we could definitely write unit tests as well.

Reply:
I also like to see the map behavior. Without seeing the zoom level or scale in the story, it is hard for me to tell if the intended behavior is matching the actual behavior. We can tackle this at a later point though. There are a lot of [specs from the layer selector we should convert to jest and the tests ](https://github.com/agrc-widgets/layer-selector/tree/master/tests) should be stories.